tests/cases/fourslash/getEditsForFileRename_tsconfig.ts

```diff
@@ -16,6 +16,9 @@
 ////    "exclude": ["old"],
 ////}
 
+// @Filename: /src/old/someFile.ts
+////
```

Comment:
With this change it seems we need to have oldFile present on the disk? What about the tests when getEditsForRename when oldFile isnt present on the disk?

Reply:
For `getEditsForFileRename` to work, either oldFile xor newFile should be present. The tests start with oldFile present, verify the output of `getEditsForFileRename`, then rename the file, and verify the output of `getEditsForFileRename` again.